tests/unit/fem/test_pa_coeff.cpp

```diff
@@ -14,6 +14,8 @@
 
 using namespace mfem;
 
+const double PI = 3.14159265358979323846;
```

Comment:
If `#include "mfem.hpp"` is before `#include "catch.hpp"` then `<cmath>` will be included after the mfem config header which enables the definition of `M_PI` by `<cmath>` under windows (one needs to `#define _USE_MATH_DEFINES` before including `<cmath>` to get constants like `M_PI`): https://github.com/mfem/mfem/blob/4828e7e2a72b707f75a8e60751cffddb89c0b621/config/config.hpp#L32-L36

Reply:
Ah, I was wondering why it wasn't working. Thanks! Just shuffled the code a bit.